### PR TITLE
feat: loading indicator for charts

### LINKED ISSUE #109: Detect mobile on portfolio page
If a user is trying to visit the portfolio page, we should have a hook that detects mobile and renders the following view if detected. See [getCompatibility()](https://github.com/penumbra-zone/web/blob/074584500b1dd143c927cadb8ceb85d0b72e5204/packages/ui/components/ui/incompatible-browser-banner/get-compatibility.ts#L16) in minifront as reference for parsing user-agent.

Screenshot 2024-10-24 at 4 41 00 PM

Reference: https://www.figma.com/design/xWsVprkEgEihweVN9isrGQ/DEX-(Incl.-UI-Lib)?node-id=3658-145100&node-type=canvas&t=NYogohDveZXB9V4J-0

### PR DESCRIPTION
**Pull request by vacekj**
fixes #109 

- Changed ChartLoadingState from a loading message to an animated SVG graphic for better user experience.
- Adjusted the rendering logic in the Chart component to display the loading state conditionally.
- Cleaned up the layout in the TradePage component by removing unnecessary classes for better responsiveness.

### CODE REVIEW

File: src/pages/trade/ui/chart.tsx [hunk 1]
```diff
@@ -7,11 +7,102 @@ import { useCandles } from '../api/candles';
 import { observer } from 'mobx-react-lite';
 import { DurationWindow, durationWindows } from '@/shared/utils/duration.ts';
 
-const ChartLoadingState = () => {
+export const ChartLoadingState = () => {
   return (
-    <div className='flex w-full h-full items-center justify-center'>
-      <div className='text-gray-500'>Loading...</div>
-    </div>
+    <svg
```
Comment (grod220) [inline src/pages/trade/ui/chart.tsx:12]: issue: this svg does not match the svg from figma. Perhaps you can reach out to artur to extract the one in the designs.

File: src/pages/trade/ui/chart.tsx [hunk 2]
```diff
@@ -7,11 +7,102 @@ import { useCandles } from '../api/candles';
 import { observer } from 'mobx-react-lite';
 import { DurationWindow, durationWindows } from '@/shared/utils/duration.ts';
 
-const ChartLoadingState = () => {
+export const ChartLoadingState = () => {
```
Comment (grod220) [inline src/pages/trade/ui/chart.tsx:10]: suggestion: could you make this fade animate (sort of like the summary component) so it's not as jarring?

**Review by grod220 — COMMENTED** [review]
A few other things:
- Please include images or video in PR descriptions or in code comments if something changes visually
- The title of this PR is not correct right?
- Please add the associated issue that this closes in PR description

File: src/pages/trade/ui/chart.tsx (hunk 2 above)
Comment (grod220) [reply src/pages/trade/ui/chart.tsx:10]: does it need the `export`?

File: src/pages/trade/ui/chart.tsx (hunk 1 above)
Comment (VanishMax) [reply src/pages/trade/ui/chart.tsx:12]: We can also just import svgs as react components now in the DEX:

```ts
import SVGIcon from './icon.svg';
```

Comment (vacekj) [reply src/pages/trade/ui/chart.tsx:12]: My apology - I somehow messed up both the branch name and the title - corrected now.

Comment (vacekj) [reply src/pages/trade/ui/chart.tsx:12]: The SVG in Figma is a masked version of the larger SVG - I took just the part that is visible. I'll let Arthur know if that's okay, or fix it if not.

Comment (vacekj) [reply src/pages/trade/ui/chart.tsx:12]: Visually it is the same svg.

File: src/pages/trade/ui/chart.tsx (hunk 2 above)
Comment (vacekj) [reply src/pages/trade/ui/chart.tsx:10]: Removed the export, and added a fade-in animation.

**Review by VanishMax — APPROVED** [review]

**Pull request merged**